conformance/pom.xml

```diff
@@ -99,9 +99,11 @@
         </dependency>
     </dependencies>
     <build>
-        <sourceDirectory>src/main/scala</sourceDirectory>
```

Comment:
Why?

Reply:
Moved to main `pom.xml`, as it was repeated in all sub-modules